slave/process-kerberos-renewal-principals/changelog

```diff
@@ -0,0 +1 @@
+
```

Comment:
Changelog must be filled with first version for the package to be built, so something like this should work:
```
perun-slave-process-kerberos-renewal-principals (3.0.0) stable; urgency=low

  * added new service kerberos-renewal-principals

 -- Johana Supikova <xsupikov@fi.muni.cz>  Wed, 11 Aug 2021 10:37:00 +0200
```